presto-native-execution/presto_cpp/main/tests/PrestoTestMain.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#include <folly/init/Init.h>
+#include <gflags/gflags.h>
+#include <gflags/gflags_declare.h>
+#include <glog/logging.h>
+#include "PrestoTestServer.h"
+#include "velox/common/base/StatsReporter.h"
+
+DEFINE_string(etc_dir, ".", "etc directory for presto configuration");
+
+int main(int argc, char* argv[]) {
+ folly::init(&argc, &argv);
+
+ facebook::presto::PrestoTestServer presto(FLAGS_etc_dir);
+ presto.run();
+ LOG(INFO) << "SHUTDOWN: Exiting main()";
```

Comment:
nit: It might be useful to qualify this with a test verb e.g. "Exiting presto test main()". The reason I think it might be useful is because currently our test is using presto_server and as we switch over to presto_test_server, having a log hint to determine the binary used could be a troubleshooting assist.

Reply:
Thanks Ankur. I updated the PR a couple of days ago. this should be fixed. :)